lib/extensions/boolean_extension.rb

```diff
@@ -3,11 +3,19 @@ module True
     def to_i
       1
     end
+
+    def to_yesno
```

Comment:
**[not a blocker]** This seems more like a representation of a boolean value for the views rather than an extension for the `Boolean` type. Also, we're doing an implicit translation here, so if we needed to have this is _Cymraeg_ or something like that it would be a bit more difficult.

I would suggest for us to use a presenter where necessary.